flink-python/pyflink/datastream/checkpoint_config.py

```diff
@@ -268,15 +268,15 @@ def get_externalized_checkpoint_cleanup(self):
         cleanup_mode = self._j_checkpoint_config.getExternalizedCheckpointCleanup()
         return ExternalizedCheckpointCleanup._from_j_externalized_checkpoint_cleanup(cleanup_mode)
 
-    def is_unaligned_checkpoints_enabled(self):
+    def is_unaligned_checkpoints_enabled(self) -> bool:
         """
         Returns whether unaligned checkpoints are enabled.
 
         :return: ``True`` if unaligned checkpoints are enabled.
         """
         return self._j_checkpoint_config.isUnalignedCheckpointsEnabled()
 
-    def enable_unaligned_checkpoints(self, enabled=True):
+    def enable_unaligned_checkpoints(self, enabled: bool = True):
         """
         Enables unaligned checkpoints, which greatly reduce checkpointing times under backpressure.
 
```

Comment:
what about converting ExternalizedCheckpointCleanup to enum type, then we could add type hint for enable_externalized_checkpoints/get_externalized_checkpoint_cleanup? This will also make it easier to use.

Reply:
Yes. Make sense